WalletWasabi.Fluent/ViewModels/Wallets/Buy/Workflows/ShopinBit/1_Initial/WelcomeStep.cs

```diff
@@ -35,17 +36,7 @@ public WelcomeStep(Conversation conversation, CancellationToken token) : base(co
 
 	protected override IEnumerable<string> BotMessages(Conversation conversation)
 	{
-		// Welcome
-		yield return "Please select the assistant that best fits your needs:";
-
-		// All-Purpose Concierge Assistant
-		yield return "All-Purpose Concierge Assistant\n\nFor a wide range of purchases, from vehicles to tech gadgets and more.";
-
-		// Fast Travel Assistant
-		yield return "Fast Travel Assistant\n\nIf you've a specific flight or hotel in mind and need quick assistance with booking.";
-
-		// General Travel Assistant
-		yield return "General Travel Assistant\n\nIf you're just starting to plan your travel and don't have any details yet.";
+		yield return "README / EXPLANATION: https://shopinbit.com/wasabiwelcome";
```

Comment:
This should be something like this:

```suggestion
		yield return $"Please select the assistant that best fits your needs. Read more about them here: {ServiceDescriptionUrl}";
```